docs/user-guide/topics/ab_testing.md

```diff
@@ -0,0 +1,203 @@
+---
+template: main.html
+---
+
+# A/B/n Testing
+
+A/B/n testing relies on business metrics typically computed by a frontend, user-facing, application component. Metric values often depend on one or more interactions with backend (not user-facing) components. To A/B/n test a backend component, it is necessary to be able to associate a metric value (computed by the frontend) to the version of the backend component that contributed to its computation. The challenge is that the frontend service often does not know which version of the backend component processed a given request.
```

Comment:
Are there figures we can include in this page? Of course, we can reuse figures from the blog article or even the tutorial if needed.

Reply:
I seem to remember the GitHub issue / design discussions having some relevant figures.